src/bgw/job.c

```diff
@@ -365,7 +367,7 @@ ts_bgw_job_get_scheduled(size_t alloc_size, MemoryContext mctx)
 
 		BgwJob *job = MemoryContextAllocZero(mctx, alloc_size);
 		HeapTuple tuple = ts_scanner_fetch_heap_tuple(ti, false, &should_free);
-		memcpy(job, GETSTRUCT(tuple), sizeof(FormData_bgw_job));
+		memcpy(job, GETSTRUCT(tuple), offsetof(FormData_bgw_job, fixed_schedule));
```

Comment:
Since we are handling the varlen struct members of `FormData_bgw_job` later, could you also change the existing comment in line 382/384 to make this more clear?

Reply:
Added a comment about the fixed-width header.